src/Stoa.ts

```diff
@@ -98,18 +98,12 @@ class Stoa {
                     return;
                 }
 
-                if (rows.length)
+                if (rows.length == 1)
                 {
-                    let out_put:Array<ValidatorData> = new Array<ValidatorData>();
-
-                    for (const row of rows)
-                    {
-                        let preimage: IPreimage = {distance: row.distance, hash: '0'} as IPreimage;
-                        var validator: ValidatorData | null =
-                            new ValidatorData(row.address, row.enrolled_at, row.stake, preimage);
-                        out_put.push(validator);
-                    }
-                    res.status(200).send(JSON.stringify(out_put));
+                    let preimage: IPreimage = {distance: rows[0].distance, hash: '0'} as IPreimage;
```

Comment:
Ah that explains it :D

Reply:
I will open another PR related to the preimage.